llvm/test/CodeGen/PowerPC/sms-regpress.mir

```diff
@@ -0,0 +1,330 @@
+# RUN: llc --verify-machineinstrs -mcpu=pwr9 -o - %s -run-pass=pipeliner -ppc-enable-pipeliner  -pipeliner-register-pressure -pipeliner-max-mii=50 -pipeliner-ii-search-range=30 -pipeliner-max-stages=10 -debug-only=pipeliner 2>&1 | FileCheck %s
+
+# Check that if the register pressure is too high, the schedule is rejected, II is incremented, and scheduling continues.
+# The specific value of II is not important.
+
+# CHECK: Try to schedule with 21
+# CHECK: 	Can't schedule
+# CHECK: Try to schedule with 22
+# CHECK: 	Can't schedule
+# CHECK: Try to schedule with 23
+# CHECK: Rejecte the schedule because of too high register pressure
+# CHECK: Try to schedule with 24
+# CHECK: Rejecte the schedule because of too high register pressure
+# CHECK: Try to schedule with 25
+# CHECK: Rejecte the schedule because of too high register pressure
+# CHECK: Try to schedule with 26
+# CHECK: Rejecte the schedule because of too high register pressure
```

Comment:
These messages need to be updated, to fix the CI error.

Reply:
Oh, sorry, I forgot. Thanks!